flocker/control/test/test_httpapi.py

```diff
@@ -2745,24 +2746,33 @@ def test_returns_service(self):
         """
         ``create_api_service`` returns an object providing ``IService``.
         """
+        path = FilePath(self.mktemp())
+        path.makedirs()
+        authority = RootCredential.initialize(path, b"mycluster")
+        ControlCredential.initialize(path, authority)
         reactor = MemoryReactor()
         endpoint = TCP4ServerEndpoint(reactor, 6789)
-        verifyObject(IService, create_api_service(None, None, endpoint))
+        verifyObject(IService, create_api_service(
+            None, None, endpoint, path))
 
     def test_listens_endpoint(self):
         """
         ``create_api_service`` returns a service that listens using the given
         endpoint with a HTTP server.
```

Comment:
This isn't tested anymore. On the other hand it's a bit of a silly test... Just fix the docstring i guess.